spec.html

```diff
@@ -41090,29 +41090,29 @@ <h1>
       </emu-clause>
 
       <emu-clause id="sec-%typedarray%.prototype.slice">
-        <h1>%TypedArray%.prototype.slice ( _start_, _end_ )</h1>
+        <h1>%TypedArray%.prototype.slice ( _startArg_, _endArg_ )</h1>
         <p>The interpretation and use of the arguments of this method are the same as for `Array.prototype.slice` as defined in <emu-xref href="#sec-array.prototype.slice"></emu-xref>.</p>
         <p>This method performs the following steps when called:</p>
         <emu-alg>
           1. Let _O_ be the *this* value.
           1. Let _taRecord_ be ? ValidateTypedArray(_O_, ~seq-cst~).
-          1. Let _len_ be TypedArrayLength(_taRecord_).
-          1. Let _relativeStart_ be ? ToIntegerOrInfinity(_start_).
-          1. If _relativeStart_ = -∞, let _k_ be 0.
-          1. Else if _relativeStart_ &lt; 0, let _k_ be max(_len_ + _relativeStart_, 0).
-          1. Else, let _k_ be min(_relativeStart_, _len_).
-          1. If _end_ is *undefined*, let _relativeEnd_ be _len_; else let _relativeEnd_ be ? ToIntegerOrInfinity(_end_).
-          1. If _relativeEnd_ = -∞, let _final_ be 0.
-          1. Else if _relativeEnd_ &lt; 0, let _final_ be max(_len_ + _relativeEnd_, 0).
-          1. Else, let _final_ be min(_relativeEnd_, _len_).
-          1. Let _count_ be max(_final_ - _k_, 0).
-          1. Let _A_ be ? TypedArraySpeciesCreate(_O_, « 𝔽(_count_) »).
-          1. If _count_ > 0, then
+          1. Let _srcArrayLength_ be TypedArrayLength(_taRecord_).
+          1. Let _relativeStart_ be ? ToIntegerOrInfinity(_startArg_).
+          1. If _relativeStart_ = -∞, let _start_ be 0.
+          1. Else if _relativeStart_ &lt; 0, let _start_ be max(_srcArrayLength_ + _relativeStart_, 0).
+          1. Else, let _start_ be min(_relativeStart_, _srcArrayLength_).
+          1. If _endArg_ is *undefined*, let _relativeEnd_ be _srcArrayLength_; else let _relativeEnd_ be ? ToIntegerOrInfinity(_endArg_).
+          1. If _relativeEnd_ = -∞, let _end_ be 0.
+          1. Else if _relativeEnd_ &lt; 0, let _end_ be max(_srcArrayLength_ + _relativeEnd_, 0).
+          1. Else, let _end_ be min(_relativeEnd_, _srcArrayLength_).
+          1. Let _countBytes_ be max(_end_ - _start_, 0).
+          1. Let _A_ be ? TypedArraySpeciesCreate(_O_, « 𝔽(_countBytes_) »).
+          1. If _countBytes_ > 0, then
             1. Set _taRecord_ to MakeTypedArrayWithBufferWitnessRecord(_O_, ~seq-cst~).
             1. If IsTypedArrayOutOfBounds(_taRecord_) is *true*, throw a *TypeError* exception.
-            1. Set _len_ to TypedArrayLength(_taRecord_).
-            1. Set _final_ to min(_final_, _len_).
-            1. Set _count_ to max(_final_ - _k_, 0).
+            1. Set _srcArrayLength_ to TypedArrayLength(_taRecord_).
+            1. Set _end_ to min(_end_, _srcArrayLength_).
```

Comment:
Not an alias rename, but I'd prefer to inline this expression on the next line and remove the re-assignment to `end` so that we have fewer changing aliases.

Reply:
Done.